test/test_translator.py

```diff
@@ -232,26 +229,10 @@ def test_get_event_status():
 def test_wzdx_creator() :
     wzdx_schema = json.loads(open('translator/sample files/validation_schema/wzdx_v3.0_feed.json').read())
     icone_data =open('translator/sample files/Icone Data/incidents_extended.xml').read()
-
-
     icone_obj = xmltodict.parse(icone_data)
-
-    # info={}
-    # info['feed_info_id'] = "feed_info_id"
-    #
-    # #### This information is required, might want to hardcode
-    # info['metadata'] = {}
-    # info['metadata']['wz_location_method'] = "channel-device-method"
-    # info['metadata']['lrs_type'] = "lrs_type"
-    # info['metadata']['location_verify_method'] = "location_verify_method"
-    # info['metadata']['datafeed_frequency_update'] = 86400
-    # info['metadata']['timestamp_metadata_update'] = "timestamp_metadata_update"
-    # info['metadata']['contact_name'] = "contact_name"
-    # info['metadata']['contact_email'] = "contact_email"
-    # info['metadata']['issuing_organization'] = "issuing_organization"
-    test_wzdx=icone_translator.wzdx_creator(icone_obj,icone_translator.initialize_info())
-
-    validate(instance=test_wzdx,schema=wzdx_schema)
+    test_wzdx = icone_translator.wzdx_creator(icone_obj, icone_translator.initialize_info())
```

Comment:
This is the only test against the `wzdx_creator()` function, which means the only case we are testing is a valid icon_obj, and valid info object. We want to test all the edge cases - what happens if we pass in a null icone_obj, or no info object for instance?

Reply:
Added some unity test edge cases for the function and pushed the updated code to dev branch.

Reply:
This is still missing edge cases such as no/invalid info object, and callback. We're also missing testing logic for missing `datafeed_frequency_update`, no incidents found, etc.

Reply:
Added test cases for invalid info objects : test_wzdx_creator_invalid_info_object
no incident found : test_wzdx_creator_no_incidents()
ni info object : test_wzdx_creator_no_info_object()